log/log.go

```diff
@@ -0,0 +1,28 @@
+package log
+
+import (
+	l "log"
+
+	raven "github.com/getsentry/raven-go"
```

Comment:
Annoying that they don't implement a log handler. :/ Don't think there's much more you can do about it tho.